frontend/packages/operator-lifecycle-manager/src/components/install-plan.tsx

```diff
@@ -37,13 +37,12 @@ import {
   SubscriptionModel,
   ClusterServiceVersionModel,
   InstallPlanModel,
-  CatalogSourceModel,
   OperatorGroupModel,
 } from '../models';
 import { InstallPlanKind, InstallPlanApproval, Step } from '../types';
 import { requireOperatorGroup } from './operator-group';
 import { installPlanPreviewModal } from './modals/installplan-preview-modal';
-import { olmNamespace, referenceForStepResource } from './index';
```

Comment:
Can we remove the `olmNamespace` constant?

Reply:
Probably, I don't think it's ever used.